lib/chef/knife/ec2_server_create.rb

```diff
@@ -67,10 +67,18 @@ class Ec2ServerCreate < Knife
         :proc => Proc.new { |groups| groups.split(',') }
 
       option :security_group_ids,
-        :short => "-g 'X,Y,Z'",
-        :long => "--security-group-ids 'X,Y,Z'",
-        :description => "The security group ids for this server; required when using VPC,Please provide values in format --security-group-ids 'X,Y,Z'",
-        :proc => Proc.new { |security_group_ids| security_group_ids.split(',') }
+        :short => "-g SECURITY_GROUP_IDS",
+        :long => "--security-group-ids",
```

Comment:
I think "deprecation" and raising an error are two different states. I think the safer approach here is to deprecate it now and raise an error in a future release. If we say it is deprecated, that means it should continue to work but we make it very clear that its lifetime is limited. @btm please comment if you have different thoughts but I don't think we can say its deprecated and disallow it at the same time.